app/src/main/webapp/app/error/error.ctrl.js

```diff
@@ -170,5 +170,54 @@ esbMessageAdminApp.controller('ErrorsSearchCtrl', [
       $scope.calendarToOpened = true;
     };
     // End Date picker stuff
+
+    // on message select, fetch message details
+    $rootScope.$watch('selectedMessage',
```

Comment:
Also, if messageSelections.length == 0, then (like line 214 now), we should set `$scope.message = null`